ui/media/js/engine.js

```diff
@@ -437,6 +437,9 @@
          * @memberof Task
          */
         async post(url, timeout=-1) {
+	    if (typeof pauseClient != 'undefined' && pauseClient===true && typeof resumeClient === 'function') {
+	        await resumeClient()
```

Comment:
The event callback onIdle is now async in main.js
https://github.com/cmdr2/stable-diffusion-ui/pull/643

Calling `await resumeClient()` from onIdle will now do just the same and would keep all the UI code inside main.js